include/tack_dummy.cpp

```diff
@@ -0,0 +1,3 @@
+// this file only exists to allow cmake to generate a valid library
+
+#include "tack/float4.h"
```

Comment:
This was intended to be a relative path, but with the suggested changes this file will disappear.